### PR TITLE
Navigation implementation

### PR DESCRIPTION
**Pull request by Ne-anna**
Implemented navigation with logo, drop-down Cart, animating hamburger which opens menu, from menu can open sub-menu. Have changed db.json file, added needed texts/paths.

### CODE REVIEW

File: src/Components/Molecules/menu/menu.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const Menu = (props) => {
+    const menu = jsonFile.menu;
+
+    return (
+        <div className={`navigation__menu`}>
+            {
+                menu.map((post, index) => {
+                    return (
+                        <li className={`navigation__item ${post.isSubMenu ? 'navigation__item--features' : post.title === "Search" ? 'navigation__item--search' : ''}`} key={index}>
```
Comment (ReinisZogots) [inline src/Components/Molecules/menu/menu.js:14]: Maybe there is a better way to structure this?

File: src/db.json [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+    {
+      "item": "View Cart"
+    },
+    {
+      "item": "Checkout"
+    }
+  ],
+  "navigation": {
+    "logo": "Renome",
+    "cart": "/assets/icons/icon-cart.svg",
```
Comment (ReinisZogots) [inline src/db.json:15]: cartPath?

File: src/db.json [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+    "altTagCart": "cart icon",
+    "count": "2",
+    "divider": "/assets/icons/icon-divider.svg",
```
Comment (ReinisZogots) [inline src/db.json:18]: dividerPath?

File: src/db.json [hunk 4]
```diff
@@ -47,6 +66,9 @@
       "url": "#"
     }
   ],
+  "wordBack": {
```
Comment (ReinisZogots) [inline src/db.json:69]: Back button title?

File: src/db.json [hunk 5]
```diff
@@ -47,6 +66,9 @@
...
+    "back": "back"
```
Comment (ReinisZogots) [inline src/db.json:70]: naming

File: src/Components/Atoms/buttons/hamburger-button/hamburger.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
+import './hamburger.scss';
+import React, { useEffect, useRef, useState } from 'react';
+import Menu from 'src/Components/Molecules/menu/menu';
+import SubMenu from 'src/Components/Molecules/sub-menu.js/sub-menu';
+
+const Hamburger = () => {
+    const [isActive, setActive] = useState(false);
```
Comment (ReinisZogots) [inline src/Components/Atoms/buttons/hamburger-button/hamburger.js:7]: does isActive means is open?

Comment (Ne-anna) [reply src/Components/Atoms/buttons/hamburger-button/hamburger.js:7]: isActive is open(hamburger X), while(X) is visible it counts as active.

File: src/App.js [hunk 7]
```diff
@@ -13,9 +14,13 @@ function App() {
 
   return (
     <div className="App">
+      <header>
+        <Navigation
+        />
```
Comment (EgijaLocmele) [inline src/App.js:19]: Don't have this in next line

File: src/Components/Molecules/sub-menu.js/sub-menu.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+const SubMenu = (props) => {
+    const menu = jsonFile.menu;
+    const back = jsonFile.wordBack;
+
+    return (
+        <div className="sub-navigation">
+            <div className="sub-navigation__item--back">
+                <div onClick={props.subMenuClose}> <SubMenuBack /></div>
+
```
Comment (EgijaLocmele) [inline src/Components/Molecules/sub-menu.js/sub-menu.js:13]: Delete empty line and move `<SubMenuBack />` to next line

File: src/Components/Molecules/sub-menu.js/sub-menu.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+                <p className="sub-navigation__text--back" onClick={props.subMenuClose}>{back.back}</p>
```
Comment (EgijaLocmele) [inline src/Components/Molecules/sub-menu.js/sub-menu.js:14]: Split this
Also, what is `back.back`? Would suggest renaming it

File: src/Components/Molecules/sub-menu.js/sub-menu.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+            </div>
+            {
+                menu.map((post => {
+                    return (
+                        (typeof (post.subMenu) == 'object') ?
+                            <>
+                                {
+                                    post.subMenu.map((post) =>
+                                        <div className='sub-navigation__item'>
+                                            <p className="sub-navigation__text"> {post.title} </p>
```
Comment (EgijaLocmele) [inline src/Components/Molecules/sub-menu.js/sub-menu.js:24]: Split this

File: src/db.json [hunk 11]
```diff
@@ -69,7 +91,7 @@
   ],
   "about": {
     "images": {
-      "top": "Assets/img/about_img_2.png",
+      "top": "/assets/img/about_img_2.png",
       "altTagBack": "white house from afar",
       "bottom": "assets/img/about_img_1.png",
```
Comment (EgijaLocmele) [inline src/db.json:96]: Doesn't the path need `/` before assets?

**Review by EgijaLocmele — APPROVED** [review]

**Review by ReinisZogots — APPROVED** [review]

**Pull request merged**